packages/strapi-plugin-users-permissions/services/Providers.js

```diff
@@ -99,169 +122,199 @@ exports.connect = (provider, query) => {
       const discord = new Purest({
         provider: 'discord',
         config: {
-          'discord': {
+          discord: {
             'https://discordapp.com/api/': {
-              '__domain': {
-                'auth': {
-                  'auth': {'bearer': '[0]'}
-                }
+              __domain: {
+                auth: {
+                  auth: { bearer: '[0]' },
+                },
               },
               '{endpoint}': {
-                '__path': {
-                  'alias': '__default'
-                }
-              }
-            }
-          }
-        }
-      });
-      discord.query().get('users/@me').auth(access_token).request((err, res, body) => {
-        if (err) {
-          callback(err);
-        } else {
-          // Combine username and discriminator because discord username is not unique
-          var username = `${body.username}#${body.discriminator}`;
-          callback(null, {
-            username: username,
-            email: body.email
-          });
-        }
+                __path: {
+                  alias: '__default',
+                },
+              },
+            },
+          },
+        },
       });
+      discord
+        .query()
+        .get('users/@me')
+        .auth(access_token)
+        .request((err, res, body) => {
+          if (err) {
+            callback(err);
+          } else {
+            // Combine username and discriminator because discord username is not unique
+            var username = `${body.username}#${body.discriminator}`;
+            callback(null, {
+              username: username,
+              email: body.email,
+            });
+          }
+        });
       break;
     }
     case 'facebook': {
       const facebook = new Purest({
-        provider: 'facebook'
+        provider: 'facebook',
       });
 
-      facebook.query().get('me?fields=name,email').auth(access_token).request((err, res, body) => {
-        if (err) {
-          callback(err);
-        } else {
-          callback(null, {
-            username: body.name,
-            email: body.email
-          });
-        }
-      });
+      facebook
+        .query()
+        .get('me?fields=name,email')
+        .auth(access_token)
+        .request((err, res, body) => {
+          if (err) {
+            callback(err);
+          } else {
+            callback(null, {
+              username: body.name,
+              email: body.email,
+            });
+          }
+        });
       break;
     }
     case 'google': {
       const google = new Purest({
-        provider: 'google'
+        provider: 'google',
       });
 
-      google.query('plus').get('people/me').auth(access_token).request((err, res, body) => {
-        if (err) {
-          callback(err);
-        } else {
-          callback(null, {
-            username: body.emails[0].value.split("@")[0],
-            email: body.emails[0].value
-          });
-        }
-      });
+      google
```

Comment:
Just a reminder, in an other PR I fixed the Google Auth provider - Please make sure to use the fix when you will merge with `master`